### ISSUE TITLE
Could not load file or assembly 'Microsoft.ML.Data, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'

### ISSUE THREAD

**Issue by BritoRon** [dotnet/machinelearning-modelbuilder#361]
When I trying to use 

            _ModelOutput result = ConsumeModel.Predict(input);_

I ran into issue

Could not load file or assembly 'Microsoft.ML.Data, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.

If anyone can give any tips I'm grateful.

**Comment by JakeRadMSFT** [comment]
Can you give us more context? Are you in VS or using the CLI?

Can you try restoring NuGets?

**Comment by BritoRon** [comment]
My environment is visual studio 2019 Community - Version 16.3.10, Microsofr .NET Framework - Version 4.8.03572.

I was following the sentiment analysis scenario tutorial from ML.NET Tutorial.
At the end of the tutorial, when asked to run the program, I ran into issue above.

I tried a lot of things, but in the end I uninstalled and reinstalled Visual Studio 2019. Now it worked until the end of the tutorial without any further problems.
